llama.cpp

```diff
@@ -1723,6 +1751,7 @@ enum e_model {
     MODEL_MEDIUM,
     MODEL_LARGE,
     MODEL_XL,
+    MODEL_A2_7B,
```

Comment:
This also needs an entry in `llama_model_type_name` for its string representation.